packages/suite-web/e2e/tests/onboarding/t2t1-create-wallet.test.ts

```diff
@@ -8,78 +8,11 @@ describe('Onboarding - create wallet', () => {
         cy.prefixedVisit('/');
     });
 
-    // cypress open todo
-    it('Success (no shamir capability)', () => {
-        cy.task('startEmu', { wipe: true, version: '2.1.1' });
```

Comment:
this test does not work. maybe trezorlib update. I dont know. but we were testing 2.1.1 which is now definitely more than 2 years old